docs/README.RaspberryPi.md

```diff
@@ -0,0 +1,136 @@
+![Kodi Logo](resources/banner_slim.png)
+
+# Raspberry Pi build guide
+This guide has been tested with x86_64 Ubuntu 16.04 (Xenial) and 17.10 (Artful). It is meant to cross-compile Kodi for the Raspberry Pi using **[Kodi's unified depends build system](../tools/depends/README.md)**. Please read it in full before you proceed to familiarize yourself with the build procedure.
+
+If you're looking to build Kodi natively using **[Raspbian](https://www.raspberrypi.org/downloads/raspbian/)**, you should follow the **[Ubuntu guide](README.Ubuntu.md)** instead. Several other distributions have **[specific guides](README.md)** and a general **[Linux guide](README.Linux.md)** is also available.
+
+## Table of Contents
+1. **[Document conventions](#1-document-conventions)**
+2. **[Install the required packages](#2-install-the-required-packages)**
+3. **[Get the source code](#3-get-the-source-code)**  
+  3.1. **[Get Raspberry Pi tools and firmware](#31-get-raspberry-pi-tools-and-firmware)**
+4. **[Build tools and dependencies](#4-build-tools-and-dependencies)**
+5. **[Build Kodi](#5-build-kodi)**
+
+## 1. Document conventions
+This guide assumes you are using `terminal`, also known as `console`, `command-line` or simply `cli`. Commands need to be run at the terminal, one at a time and in the provided order.
+
+This is a comment that provides context:
+```
+this is a command
+this is another command
+and yet another one
+```
+
+**Example:** Clone Kodi's current master branch:
+```
+git clone https://github.com/xbmc/xbmc kodi
+```
+
+Commands that contain strings enclosed in angle brackets denote something you need to change to suit your needs.
+```
+git clone -b <branch-name> https://github.com/xbmc/xbmc kodi
+```
+
+**Example:** Clone Kodi's current Krypton branch:
+```
+git clone -b Krypton https://github.com/xbmc/xbmc kodi
+```
+
+Several different strategies are used to draw your attention to certain pieces of information. In order of how critical the information is, these items are marked as a note, tip, or warning. For example:
+ 
+**NOTE:** Linux is user friendly... It's just very particular about who its friends are.  
+**TIP:** Algorithm is what developers call code they do not want to explain.  
+**WARNING:** Developers don't change light bulbs. It's a hardware problem.
+
+**[back to top](#table-of-contents)** | **[back to section top](#1-document-conventions)**
+
+## 2. Install the required packages
+Install build dependencies needed to cross-compile Kodi for the Raspberry Pi:
+```
+sudo apt install autoconf build-essential cmake curl gawk git gperf lib32stdc++6 lib32z1 lib32z1-dev libcurl4-openssl-dev zlib1g-dev
```

Comment:
lib32* stuff is not needed anymore, rpi toolchain from https://github.com/raspberrypi/tools is 64bit,
```
asavah@kodi-dep:~/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin$ file arm-linux-gnueabihf-gcc
arm-linux-gnueabihf-gcc: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.24, BuildID[sha1]=e0df943f126fe91df411d950dd26652126a1e06d, stripped
asavah@kodi-dep:~/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin$ ldd arm-linux-gnueabihf-gcc
        linux-vdso.so.1 =>  (0x00007ffca53fb000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007facfbd45000)
        /lib64/ld-linux-x86-64.so.2 (0x00007facfc10f000)
```